src/azul/drs.py

```diff
@@ -185,6 +167,28 @@ def drs_object(object_uuid, version=None):
     }
 
 
+class DRSObject:
```

Comment:
If you go to the trouble of adding a class, you should add type annotations on method signatures and attributes. And use `@dataclass` or `NamedTuple`.